src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const L0NG_TERM_DISCOUNT = 50;
```

Comment:
There is a typo in the constant name `L0NG_TERM_DISCOUNT`. It should be `LONG_TERM_DISCOUNT` to maintain consistency and avoid confusion.